atlp-rwanda/hackers-ec-Fe
# #187419006 Implement Reset Password

#### What does this PR do?
This PR adds a feature that allows users to reset their forgotten passwords.

#### Description of Task to be completed?
- User click on "Forgot password" and get email which contain token for reseting password.
- After to click on it, its redirect to page where user enter his/her password along with confirm password in form with validation.

#### How should this be manually tested?
1. Navigate to the login page.
2. Click on the 'Forgot Password?' link.
3. Enter the associated email address and submit.
4. Check the email for the reset password token.
5. Follow the link to the reset password page.
6. Enter and submit the new password.

#### What are the relevant pivotal tracker/Trello stories?
#187419006
